doc/sphinx/proof-engine/tactics.rst

```diff
@@ -2819,19 +2819,12 @@ simply :g:`t=u` dropping the implicit type of :g:`t` and :g:`u`.
       only in the conclusion of the goal. The clause argument must not contain
       any ``type of`` nor ``value of``.
 
-   .. tacv:: cutrewrite <- (@term = @term’)
+   .. tacv:: cutrewrite {| <- | -> } (@term = @term’)
```

Comment:
The full syntax is `cutrewrite {? {| -> | <- } } @one_term {? in @ident }`, with an apparent semantic restriction that @one_term is in the form `(@term__1 = @term__2)`